frontend/src/Pages/AllUser/Login/ResetPassword.jsx

```
@@ -0,0 +1,167 @@
+                            required
+                        />
+                    </div>
+
+                    {/* Custom Checkboxes for Password Validations */}
+                    <div className="space-y-3 mt-4">
+                        {/* Minimum Length Validation */}
+                        <div className="flex items-center space-x-3">
+                            <div className={`h-5 w-5 rounded-full border-2 flex items-center justify-center 
+            ${isMinLengthValid ? 'border-green-500 bg-green-500' : 'border-gray-400'}`}>
+                                {isMinLengthValid && (
+                                    <svg className="w-3 h-3 text-white" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)